### DISCUSSION TITLE
Difference between VertiPaq model size and PBIX file size

### DISCUSSION THREAD

**Discussion by ajerb** [sql-bi/Bravo#528] [Q&A] [answered]
### Describe the problem

Hi,
I am surprised to see the difference my Dataset size that Bravo displays VS the size that is displayed in the service or in my desktop

 ![image](https://user-images.githubusercontent.com/44672352/199777234-4fdb8493-d7c3-41a4-bd24-a517ceca261a.png)

Is there an explanation to that or is it a bug ?

_No response_

**Comment by albertospelta** [comment, marked as answer]
Hi @ajerb

The PBIX file on your desktop and the dataset in the service storage are both a compressed backup of your data model. When you open the PBIX report or a report on PBI service then the data model is loaded in memory in the Analysis Services process. Data loaded into memory takes more space because part of the data is not compressed so it's normal that this size - which is the one that Bravo displays - is bigger than the PBIX file on disk.

You can take a look at this video where Marco explains it in more detail https://www.youtube.com/watch?v=vI7kVVHxsiM

**Comment by ajerb** [comment]
Hi @albertospelta 
Thanks a lot, the video helped me a lot to understand it.
